psydac/fem/vector.py

```diff
@@ -304,6 +312,17 @@ def integral( self, f ):
     # Other properties and methods
     #--------------------------------------------------------------------------
     @property
+    def is_multipatch(self):
+        return False
+
+    @property
+    def is_vector_valued(self):
+        # question (MCP 03.2025) do we allow scalar-valued VectorFemSpaces ?
+        return True
```

Comment:
These two properties should be moved up, to the section "Abstract interface: read-only attributes".

Reply:
Done in 55fd2c9